work_items/revocation_methods_for_verifiable_credentials_.md

```diff
@@ -0,0 +1,53 @@
+# Revocation Methods for Verifiable Credentials
+
+The goal is to identify and define revocation methods for verifiable credentials and create the protocol to ensure interoperability.
+In this work item, possible revocation methods should be collected and assessed. Suitable revocation methods should be defined in a revocation protocol. 
+
+Tasks:
+- Collect possible revocation methods
+  - Cryptographic accumulator suggestion by Mike (https://hackmd.io/O4c3wiLZQLeXuXirm7dl9A#Math-explaination)
+  - tbd
+
+- Define an assessment setup (#of credentials, #of revocations per epoch,...)for performance testing
+- Define requirements for privacy (correlation (linked, linkable), traceability, leaking of usage data,..)
+- Define performance requirements (computational effort, storage, data transmission, sizes,...)
+- Assess the revocation methods and define one or more revocation methods that fits the purpose
+- Create the protocol 
+
+## Work Item Owners
+- Andreas Freitag (andreas.freitag.77@gmail.com)
+- Mike Lodder (redmike7@gmail.com)
+
+## External Partners (OPTIONAL)
+**This is only for work items intended for transition to an external standards
+body after incubation here. This may be filled in later and is not required for
+initial approval of the work item.**:
+- W3C
```

Comment:
```suggestion
- IETF
```

Reply:
I think this work is more suitable for Internet-level engineering. It is no more a web technology than digital signatures are.

Reply:
Not sure if we have no overlap to W3C. I would suggest to keep it in the list.

Reply:
@coffeethecup I'm ok with making the list both W3C and IETF